posts/2023-05-31-adding-new-task-to-cylc.md

```diff
@@ -0,0 +1,150 @@
+# Adding a new task to a rose suite
+
+## Issue
+In this example, we want to add an additional task to an ACCESS-CM2 run, to be executed before every instance of the coupler to update the sea surface temperature ancillary file.
+
+## Step 0: Make a copy of the ACCESS-CM2 suite
+Run this command on accessdev to make a copy of the **ACCESS-CM2** suite:
+```bash
+rosie copy u-br565
+```
+
+This command will create a copy of the original ACCESS-CM2 suite with a new suite-id that we can modify and track without interfering with the original suite.
+
+"Suite Directory" from now on will mean the directory `${HOME}/roses/<suite-id>`.
+
+## Step 1: Make a shell script to update the SST files:
+Create a file called `update_sst.sh` and located in the directory `apps/update_sst/bin` under your rose suite directory:
```

Comment:
"Create a file called `update_sst.sh` in the directory `apps/update_sst/bin" remove "and located"